code/modules/sanity/sanity_mob.dm

```diff
@@ -390,7 +394,11 @@ GLOBAL_VAR_INIT(GLOBAL_INSIGHT_MOD, 1)
 				LAZYAPLUS(stat_change, pick(ALL_STATS_FOR_LEVEL_UP), 3)
 
 			for(var/stat in stat_change)
-				owner.stats.changeStat(stat, stat_change[stat])
+				if((owner.stats.getStat(stat)) >= STAT_VALUE_MAXIMUM)
+					to_chat(owner, SPAN_NOTICE("You can not increase [stat] anymore with simple resting."))
```

Comment:
@Trilbyspaceclone I also pointed this but it was probably faded away
`You can not increase [stat] anymore with simple resting.`
into
`You feel that the rest doesn't fulfill your [stat] anymore for today.`

~~I also want to point out that this mere PR nerfs psions, self-surgery, but no one really cares about it i guess.~~